packages/client/hmi-client/src/components/responsive-matrix/cell-line-graph.vue

```diff
@@ -225,20 +227,50 @@ export default {
 				.attr('viewBox', `0 0 ${width} ${height}`)
 				.style('background', 'white');
 
-			const xAxis = axisBottom(this.xScale).tickFormat(this.labelColFormatFn);
-
-			this.svg
+			const xAxisGen = axisBottom(this.xScale).tickFormat(this.labelColFormatFn);
+			const xAxis = this.svg
 				.append('g')
 				.attr('transform', `translate(${leftMargin},${height - bottomMargin})`)
-				.call(xAxis);
-
-			const yAxis = axisLeft(this.yScale).tickFormat(this.labelRowFormatFn);
-
-			this.svg.append('g').attr('transform', `translate(${leftMargin},${topMargin})`).call(yAxis);
+				.call(xAxisGen);
+			formatAxis(xAxis);
 
-			Object.keys(this.selectedCells).forEach((parameter) => {
+			const yAxisGen = axisLeft(this.yScale).tickFormat(this.labelRowFormatFn);
+			const yAxis = this.svg
+				.append('g')
+				.attr('transform', `translate(${leftMargin},${topMargin})`)
+				.call(yAxisGen);
+			formatAxis(yAxis);
+
+			const numRows = 4;
+			const rowSize = 15;
+			const colSize = 95;
+			Object.keys(this.selectedCells).forEach((parameter, i) => {
 				this.renderLine(this.svg, parameter, this.selectedCells[parameter], this.labelColSelected);
+
+				this.svg
+					.append('rect')
+					.attr('x', 50 + colSize * Math.floor(i / numRows))
+					.attr('y', 30 + (i % numRows) * rowSize)
+					.attr('width', 8)
+					.attr('height', 8)
+					.attr('fill', this.colorFn(parameter));
+
+				this.svg
+					.append('text')
+					.attr('x', 60 + colSize * Math.floor(i / numRows))
+					.attr('y', 38 + (i % numRows) * rowSize)
+					.attr('font-size', '80%')
+					.style('fill', '#333')
```

Comment:
you need to import [Uncharted Design Tokens](https://github.com/unchartedsoftware/design-tokens/blob/main/build/js/es6/uncharted-tokens.js)

```ts
import { UnColorBlack80 } from '@node_modules/@uncharted.software/design-tokens/build/js/es6/uncharted-tokens.js';
```

```suggestion
					.style('fill', UnColorBlack80)
```

Reply:
let's handle this later since this is more or less temporary code